src/blocks/blocks/lottie/index.js

```diff
@@ -32,3 +36,30 @@ registerBlockType( name, {
 	edit,
 	save
 });
+
+const lottieAdvancedControls = createHigherOrderComponent( BlockEdit => {
```

Comment:
@arinaturcu I don't think you need to use filters for that. You should be able to use `InspectorAdvancedControls` straight where you use InspectorControls.

Reply:
Oh, that's right.